tests/machine.test.js

```diff
@@ -484,6 +484,8 @@ describe(OlmMachine.name, () => {
             expect(requests[0]).toBeInstanceOf(ToDeviceRequest);
             expect(requests[0].event_type).toEqual("m.room.encrypted");
             expect(requests[0].txn_id).toBeDefined();
+            expect(requests[0].id).toBeDefined();
+            expect(requests[0].id).toEqual(requests[0].txn_id);
```

Comment:
instead of hard-coding this into the test, can we call `markRequestAsSent` and ensure that the request doesn't come back again when you call `outgoingMessages`?

Reply:
udpated